ocs_ci/helpers/disruption_helpers.py

```diff
@@ -93,6 +103,31 @@ def set_resource(self, resource, leader_type="provisioner"):
         if self.resource == "operator":
             self.resource_obj = pod.get_operator_pods()
             self.selector = constants.OPERATOR_LABEL
+        if self.resource == "ocs_operator":
+            self.resource_obj = [pod.get_ocs_operator_pod()]
+            self.selector = constants.OCS_OPERATOR_LABEL
+        if self.resource == "alertmanager_managed_ocs_alertmanager":
+            self.resource_obj = pod.get_alertmanager_managed_ocs_alertmanager_pods()
+            self.selector = constants.MANAGED_ALERTMANAGER_LABEL
+        if self.resource == "ocs_osd_controller_manager":
```

Comment:
Can you use `elif`?

Reply:
@OdedViner 
The current structure of conditional statements was followed when adding the new pod types. Would you like to change the previous ```if``` conditions to ```elif``` ?

Reply:
got it... I dont think you need to change it `elif`..